community-operators/planetscale/planetscale-operator.v0.1.7.clusterserviceversion.yaml

```diff
@@ -0,0 +1,314 @@
+                    ]
+                  }
+                ]
+              }
+            ]
+          }
+        }
+      ]
+    categories: "Database"
+    certified: "False"
+    description: PlanetScale Vitess is a database clustering system for horizontal scaling of MySQL.
+    containerImage: registry.connect.redhat.com/planetscale/operator:0.1.7
+    createdAt: 2019-02-06T20:00:00Z
+    support: PlanetScale
+spec:
+  displayName: PlanetScale Operator
+  description: |-
+    # PlanetScale's Vitess Operator
+
+    Vitess is a database clustering system for horizontal scaling of MySQL
+    through generalized sharding.
+
+    By encapsulating shard-routing logic, Vitess allows application code and
+    database queries to remain agnostic to the distribution of data onto
+    multiple shards. With Vitess, you can even split and merge shards as your needs
+    grow, with an atomic cutover step that takes only a few seconds.
+
+    Vitess has been a core component of YouTube's database infrastructure
+    since 2011, and has grown to encompass tens of thousands of MySQL nodes.
+
+  keywords: ['mysql', 'vitess', 'sharding', 'database']
+  icon:
+  - mediatype: image/png
+    base64data: iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAFA0lEQVR42tWSf0yUdRzHH8hRJkksQeKnCHfcgXf8hotODjDgODlx6QS54+T4qSAUP44QUCDgAgVBk8EARZu10Agl8ke1LPEPZmVtNtsk14+1/sDammPU3T3P8+nzPPjVW0E3VKg+22ufP+57e7/e3+dL/efHPbPxgf+bUHvM9qGw9st2SDxiROrvM97sV9Ln8bAFQrLr3NyeT2tB6q0wInGIHZGIQ35F6Hsc+BS8drSeJbfwoO3XVxwZcpWlAobRVvyCKP56C6cRmGUcQts+NntmtQCGr1+oBIaTHaWo6gWXqBQGA8GKt5F74WTLkGleoGMcglsu0h6zAtfdtzfY8RLbbUvEY7iVyDVFVQ8nwFqF30HCKBzcf3sPbxABSeOYxUPbzKIAjRTzAhrbAhvut89HaIWhh0UB6/Y9cz1Esh0RCEeBoL0jKNAEGA54A5+5bq1xt5lOJOqOrcbwCQRiDd0WK4EZxGG+9mRXhndcAXHNEO2hmRXwLehkw1vO6csBqNC6oXmD9edvU4BnYqoHs2Kr+4ETWF/RbXKJVhEB/Zzh1hLIivCDV74NqHoT3DVNrEd6rSW4+iQd2399Mv7k98622qv7JpxltSe/jig9xAvIK478cVfgc+Tx1ST8HwTsIg5N5ArKT7Ae22osazOqzfLeLxnZwcsg0DUYuHPeqQVztucm853v9kh3HwZxXiujqB5g5OWvcwIWDN/Gv6MYFLA18rd+dvXVt37iqcqDiKZRs6z9kiW4ctDik1bEYvi8t1A0bnJRD1w1C/XNIMprhdDdXSZ52WHTqkjlJRRYYTPY+i14JuuKhLr66ai2D8ySl3rZdaU9Fp9NuwAFTpGza1RFVGz/NxSZvA9/OxNVfZwR5b4G4vw2XiK6tHPKOSR+C/n2CxppxbGrGAySsn4I3NXFCXA3YEJiZj9FPhV7dFZAMzYVpxn5YUaob2G5cELgjvpzCwq1/saSl/vkCCeAbdos3uqd3A1wDCMO5LyP8cry9NGpkfi2MTPfvmA//wZEW0tBpNL5kvYLHSJxWlI2AMLsJhQoJAIzSAI5E9N/Q7ll+Kc7+PgYvnluCyNKKwBRitZoO8G2gJe0bIARaOoYLpywZnPxsO+LpcvxEzypGLz5fmLPBATt7GDF+iZapM5hRUrNFAqsFKdoHyqcwvaPScsHWtemGzA4n/FLNzDiwv20tKwXRNmNcsXxyeT4E5MQXX/KJNbW0KKNOlqUouHalyD2ASjw0BO0qytQXHDgVrDhKBPR+C4ja/8Igiv6QZBZ8xW2vyHv+YIVZ9VN+yVmmBDaPzHjWqAqy416lBNlHOuMbj3PIMDtoKJORpBZCxGvnjWFGy+wztFqemVoIouYnMKSSh5Z8HPGC/zG0GeR2whENp1huFcu1O6FAH0zu65ykHaUxMFTIS9w3HyUxUk42XsQCN83xAToGlBgHwg1tYy7MtfiKE0gAsncWdzUYojYI7dCXzkBQm0dK0g3WPw3FTKrZGrWMXgDFz6ymOH8jmwe3SAp7QbB5mKLQJ3H+qv08HSEkgunEZ9FEyASfsrsJ4Rphe8JUnNAkJoLfqpsuNu+DVmGUIs+go05Sgz/HQHvxEx2hTThRwwWU0s1GOyAjArVeeAWu8WMAvvJb0spEYcCM89Ep04tE8qcyLdfmnBVLr9R4KJTaFLJ0oUTAfWsgE+S1oty8rKn/o3xVer4Tdr/b+dP9GyXZ3HEoQMAAAAASUVORK5CYII=
```

Comment:
@robszumski The icon I use here is from the vitess.io vitess operator. It's blury, so I was planning on switching to using the svg version of the icon. Does PlanetScale have a prefered logo for their operator?